src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReplacementService.cs

```diff
@@ -800,18 +812,25 @@ private static FilterReplacementViewModel CreateNewlyIntroducedFilterReplacement
         {
             await ReplaceFootnoteSubject(replacementPlan.Id, originalSubjectId, replacementSubjectId);
 
-            await replacementPlan.Filters.ForEachAsync(async plan =>
-                await ReplaceFootnoteFilter(replacementPlan.Id, plan));
+            await replacementPlan.Filters
```

Comment:
Here's a prime example of where our own ForEachAwaitAsnc method would cut out a few boilerplate lines.